components/ide-service/pkg/server/server.go

```diff
@@ -358,7 +358,7 @@ func (s *IDEServiceServer) ResolveWorkspaceConfig(ctx context.Context, req *api.
 						resp.IdeImageLayers = append(resp.IdeImageLayers, ide.Image)
 						template := `
 echo 'warming up stable release of ${key}...'
-JETBRAINS_BACKEND_QUALIFIER=stable /ide-desktop/jb-launcher warmup ${key}
```

Comment:
It is not good it is going to perform it for install plugins too. It should be done in the launcher specifically for actual start.

Reply:
Ok, we've moved it to launcher/main.go [[1](https://github.com/gitpod-io/gitpod/pull/15344/files#diff-ca4e002db8a555b20bbb80e8cfc4563e0aa4eea01e2a9f79340b46058c3397f1)].